tests/files.ts

```diff
@@ -1,14 +1,147 @@
-import { failUnauthenticated, test } from "./_utilities";
+import { state } from "./_cache";
+import { failUnauthenticated, test, testOperation } from "./_utilities";
 
 test.before(failUnauthenticated);
+test.serial(
+	"getFileWithoutData",
+	testOperation,
+	"getFiles",
+	() => ({
+		statusCode: 200,
+		parameters: {
+			tag: "gallery"
+		}
+	}),
+	(t) => {
+		const { body } = t.context;
+		t.is(body.length, 0, "Should have no files");
+	}
+);
+
+test.serial(
+	testOperation,
+	"createFile",
+	() => ({
+		statusCode: 200,
+		parameters: {},
+		requestBody: {
+			name: "string",
+			mimeType: "image/jpeg",
+			extension: "A",
+			tags: ["gallery"]
+		}
+	}),
+	(t) => {
+		const { body } = t.context;
+		state.set("fileId", body.id);
+		t.is(body.versions.length, 1, "File should have 1 version, the initial one");
+	}
+);
+
+test.serial(
+	"getFileWithData",
```

Comment:
Same as above, or if this is the primary successful response, we can exclude the title entirely.

Reply:
Fixed, thanks for the quick review!